aviary/validation_cases/benchmark_tests/test_0_iters.py

```diff
@@ -63,6 +63,4 @@ def test_zero_iters_solved(self):
 
 
```

Comment:
Please keep these.

Note: testflo doesn't require `unittest.main()`.  We only need that if we run with unittest, or if we want the test for run with `python test_0_iters.py`.  You can't debug through unittest though, which is why I like to keep the other lines.

Reply:
Changed back!